egret/parsers/prescient_dat_parser.py

```diff
@@ -379,11 +382,9 @@ def nd_gen_init(m,b):
     model.NondispatchableGeneratorsAtBus = Set(model.Buses, initialize=nd_gen_init)
     
     def NonNoBus_init(m):
```

Comment:
"NonNoBus" is an absolutely awful name - I would never have inferred it without the code below (and no, you didn't do this).

Reply:
Changed to `nondispatchable_generator_init`.